src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -52,12 +81,65 @@ function epidemicModelReducer(
 
 function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
   const [state, dispatch] = React.useReducer(epidemicModelReducer, {
-    stateCode: "US",
-    totalIncarcerated: populationAndHospitalData.US.incarceratedPopulation,
+    stateCode: "US Total",
+    countyName: "Total",
+    countyLevelDataLoading: true,
     rateOfSpreadFactor: 3.7,
     usePopulationSubsets: false,
   });
 
+  // fetch from external datasource
+  React.useEffect(() => {
+    fetch(
+      "https://docs.google.com/spreadsheets/d/e/2PACX-1vSeEO7JySaN21_Cxa7ON_x" +
+        "UHDM-EEOFSMIjOAoLf6YOXBurMRXZYPFi7x_aOe-0awqDcL4KZTK1NhVI/pub?gid=" +
+        "1836987932&single=true&output=csv",
+    ).then(async (response) => {
```

Comment:
Do we need an explicit `error` callback in case the fetch fails?

Reply:
yes probably, I was just cutting corners. Let me put in a quick "sorry everything failed" state as a fallback here